.gitignore

```diff
@@ -192,6 +192,7 @@ ClientBin/
 *.dbmdl
 *.dbproj.schemaview
 *.pfx
+!UT-cert.pfx
```

Comment:
Can we create the certificate in UT at runtime?

Reply:
I will try it